app/views/shared/_navbar.html.erb

```diff
@@ -30,10 +30,11 @@
               <%= link_to "My Bookings", reservations_path, class: "nav-link" %>
             </li>
             <li class="nav-item dropdown">
-              <%= image_tag "https://www.pinpng.com/pngs/m/341-3415688_no-avatar-png-transparent-png.png", class: "avatar dropdown-toggle", id: "navbarDropdown", data: { bs_toggle: "dropdown" }, 'aria-haspopup': true, 'aria-expanded': false %>
+              <%= image_tag "https://res.cloudinary.com/dvbzytnxf/image/upload/v1660835473/Hollowknight_lzisdj.webp", class: "avatar dropdown-toggle", id: "navbarDropdown", data: { bs_toggle: "dropdown" }, 'aria-haspopup': true, 'aria-expanded': false %>
               <div class="dropdown-menu dropdown-menu-end" aria-labelledby="navbarDropdown">
-                <%= link_to "Profile", "/", class: "dropdown-item" %>
-                <%= link_to "Log out", destroy_user_session_path, class: "dropdown-item", data: {turbo_method: :delete} %>
+                <%= link_to "Profile", "/", class: "dropdown-item text-center mt-1" %>
+                <%= link_to "add world", new_game_world_path, class: "dropdown-item text-center mt-1" %>
```

Comment:
the a of Add world should be a capital a. Add world.

Reply:
Good job man!

Reply:
it's fixed !